reactioncommerce/reaction
# [Marketplace] Fix notification page not found error #2614

Resolves issue #2614 

**NB:** This PR fixes the notification page not being found bug. This happened because the prefix given to the path for the `Link to` was wrong. 
It now works for both single-shop and multiple-shops scenarios, clicking the `View All` link on the notification icon will direct you to the `Notifications Page`.

**How To Test**
- Make an order as an admin or guest.
- Click on the Notifications Icon on the navbar
- Click on `View All`
- Observe that you are directed to the `notifications page`
- For multiple-shops scenario repeat the steps on the second shop.
